packages/compass-workspaces/src/stores/workspaces.ts

```diff
@@ -53,6 +59,7 @@ export const cleanupLocalAppRegistries = () => {
 
 export enum WorkspacesActions {
   OpenWorkspace = 'compass-workspaces/OpenWorkspace',
+  OpenFallbackWorkspace = 'compass-workspace/OpenFallbackWorkspace',
```

Comment:
Drive-by change: this should've always been a separate event, I was a bit lazy when adding this logic initially